src/vivarium_gates_nutrition_optimization_child/components/fertility.py

```diff
@@ -51,16 +51,13 @@ def _get_birth_records(builder: Builder) -> pd.DataFrame:
         Method to load existing fertility data to use as birth records.
         """
         data_directory = Path(builder.configuration.input_data.fertility_input_data_path)
-        scenario = builder.configuration.intervention.scenario
+        scenario = builder.configuration.intervention.parent_scenario
```

Comment:
You have a mismatch between this and what you've defined in the model-spec and branches files where you've called this `maternal_scenario`.

Reply:
Oh wow I was sure I updated this. Fixed.